torch/onnx/_internal/exporter/_registration.py

```diff
@@ -61,18 +62,9 @@ def _get_overload(qualified_name: str) -> torch._ops.OpOverload | None:
     if namespace == "math":
         return getattr(math, op_name)
     if namespace == "torchvision":
-        try:
-            import torchvision.ops  # type: ignore[import-untyped]
-        except ImportError:
+        if importlib.util.find_spec("torchvision.ops") is None:
```

Comment:
```suggestion
        if importlib.util.find_spec("torchvision") is None:
```

thanks for this change